PhysicsTools/NanoAOD/python/genparticles_cff.py

```diff
@@ -33,6 +33,9 @@
     src = cms.InputTag("finalGenParticles"),
     name= cms.string("GenPart"),
     doc = cms.string("interesting gen particles "),
+    externalVariables = cms.PSet(
+        iso = ExtVar(cms.InputTag("genIso"), float, precision=8, doc="Isolation for leptons"),
```

Comment:
@bonanomi Could you please add this new variable to [nanoDQM](https://github.com/cms-sw/cmssw/blob/7452562c868a1b4c00a2a38d8144d14bfea34262/PhysicsTools/NanoAOD/python/nanoDQM_cfi.py#L342-L354)?

Reply:
@hqucms done!